.gitignore

```diff
@@ -24,5 +24,6 @@ build/
 *.swp
 
 # HTML/PDF output from documentation
+html/
 .html
```

Comment:
Could you change these to be *.html and *.pdf? I messed up, and as is they won't ignore anything!